package.json

```diff
@@ -46,7 +47,7 @@
   "dependencies": {
     "lodash": "^4.17.21",
     "randexp": "^0.5.3",
-    "yup": "^0.29.1"
+    "yup": "^0.30.0"
```

Comment:
todo: Just to make it clear, @gabrielseco and I had a huddle last week: Before we merge this PR, ideally we'd like our internal codebase to also use Yup 0.30 so we don't have Yup twice in our bundles. 

Only then we'd merge this PR. WDYT @brennj?

Reply:
Makes sense!

Reply:
I have a [MR](https://gitlab.com/remote-com/employ-starbase/dragon/-/merge_requests/17451) in our internal repo.

My plan is to merge this and create a release, what do you think @sandrina-p

Reply:
@gabrielseco , I'm answering your question here because Github doesn't support threads is comments detached from code.

> I'll create a minor version because we're updating the yup version.  
> If it was only the fix probably, I'd do a patch.  
> What do you think @sandrina-p @brennj?

I agree!